src/main/java/org/sert2521/gamename/Log.kt

```diff
@@ -0,0 +1,45 @@
+package org.sert2521.gamename
+
+import edu.wpi.first.wpilibj.Preferences
+
+/** Checks if a log is loggable with the provided log level from SmartDashboard. */
+private fun isLoggable(level: LogLevel) =
+        level >= when (Preferences.getInstance().getString("log_level", "info")) {
+            "verbose" -> LogLevel.VERBOSE
+            "debug" -> LogLevel.DEBUG
+            "warn" -> LogLevel.WARN
+            "error" -> LogLevel.ERROR
+            else -> LogLevel.INFO
+        }
+
+enum class LogLevel {
+    /** Priority constant for the log method; use Log.v. */
+    VERBOSE,
+    /** Priority constant for the log method; use Log.d. */
+    DEBUG,
+    /** Priority constant for the log method; use Log.i. */
+    INFO,
+    /** Priority constant for the log method; use Log.w. */
+    WARN,
+    /** Priority constant for the log method; use Log.e. */
+    ERROR
```

Comment:
> [...] use Log.v.
> [...] use Log.d.
> etc.

These should be changed to whatever we eventually decide the function names should be.